docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+
+services:
+  nlisim:
```

Comment:
I'm not sure we need a `docker-compose.yml` file at all, for just 1 container which is expected to terminate (i.e. not a daemon). If we want downstream users to just run the simulation, they can easily fetch a published image via `docker pull` (or automatically with `docker run`, but if they have to use Docker Compose, then they have to find and download this file manually.

It might be a better alternative to just document the series of expected volume mounts as part of a full example `docker run -v ...` command line.

Reply:
The docker compose file is removed. Instead, we use `docker run` followed by a list of share volume to run the container. The readme has been updated for an example.